tailwind.config.js

```diff
@@ -26,10 +25,18 @@ module.exports = {
         sans: ['Bliss'],
         serif: ['Abraham']
       }
+    },
+    screens: {
```

Comment:
I recall I was the one who added these custom breakpoint values but now I realise that it will cause a lot of issues to other branches so it's better to use the default (and thus, standard) breakpoint values.

I'll revert it and redo the responsiveness (remind me if I forget plz 😓)